doc/docs/connecting.md

```diff
@@ -10,19 +10,21 @@ To start using Serial Terminal:
     ![Select Device window](./screenshots/serial_term_select_device.png "Serial Terminal Select Device view")
 
     !!! note "Note"
-         Depending on the application firmware on the device, you might see J-Link in place of the product name.
+         Depending on the application firmware on the device, you might see "J-Link" in place of the product name.
 
 1. Optionally, toggle **Auto-reconnect** on or off depending if you want Serial Terminal to automatically select the last connected serial port and attempt to connect to that port again in the future.
 
-1. Click the device you want to use.</br>
-   If the device has more than one port, Serial Terminal selects the serial port with the lowest virtual serial port index. See [Selecting a serial port](selecting_serial_port.md) for more information. You are free to select another serial port.
+1. Click the device entry to select the device.</br>
+   Serial Terminal selects the serial port with the lowest virtual serial port index (if the device has more than one port) or the previously used serial port (if you connected the device to Serial Terminal before). See [Selecting a serial port](selecting_serial_port.md) for more information about the serial port naming conventions.
```

Comment:
```suggestion
   Serial Terminal will automatically connect to the the previously used serial port if you connected the device to Serial Terminal before. See [Selecting a serial port](selecting_serial_port.md) for more information about the serial port naming conventions.
```

@greg-fer this change up to you, but I'll just explain what I mean by my edit...
1) "...serial port with the lowest virtual serial port index..." - this is just how any html select works, don't think we should explain this in the doc
2) it will auto connect to the previously used serial port and this action is slightly different from just selecting in this context

Reply:
Feels like this step tries to cover so many details... maybe just keep it small and simple, as "select the device and then serial port if needed"?
I see that auto reconnect is covered in the next steps, it just happens at the same time as selecting previously used serial port, so I wouldn't split these...